dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/package.scala

```diff
@@ -0,0 +1,22 @@
+package io.joern
+
+import io.shiftleft.codepropertygraph.generated.nodes.{Declaration, Expression, Identifier, Literal}
+import io.shiftleft.semanticcpg.language._
+import overflowdb.traversal.Traversal
+
+package object dataflowengineoss {
+
+  def globalFromLiteral(lit: Literal): Traversal[Expression] = lit
+    .where(_.inAssignment.method.name("<module>"))
```

Comment:
In JavaScript the equivalent would be `:program` so perhaps the pattern should be something like `(<module>|:program)`

Reply:
I'll make it `.nameExact("<module>", ":program")`, that's more exact.